packages/lib/serverConfig.ts

```diff
@@ -13,10 +13,12 @@ function detectTransport(): SendmailTransport.Options | SMTPConnection.Options |
     const transport = {
       host: process.env.EMAIL_SERVER_HOST,
       port,
-      auth: {
-        user: process.env.EMAIL_SERVER_USER,
-        pass: process.env.EMAIL_SERVER_PASSWORD,
-      },
+      auth: process.env.EMAIL_SERVER_PASSWORD
```

Comment:
I think a better approach here would be to use `isENVProd` check instead of the server password. Would make the use case more clearer